filter:
  max_gaussians: 100, # limit on number of Gaussians
  prune_threshold: .00001,  # pruning threshold
  merge_threshold: 4.0, # merging threshold
  p_gate: 0.999, #gate size in percentage
  gating: true #enable/disable gating